intuita-webview/src/jobDiffView/DiffViewer/Container.tsx

```diff
@@ -65,7 +67,11 @@ export const Container = forwardRef<HTMLDivElement, ContainerProps>(
 );
 
 type HeaderProps = Readonly<{
+	diffChanges: DiffChanges | null;
```

Comment:
I would call this structure `DiffChange` or `Diff`, the reason is that plural usually means the structure is an collection.